reference-adapter/adapter-master-data-service/src/main/resources/db/migration/main/V202301161027__adapter_master_data_ddl.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE department
+(
+    id character varying(255)  NOT NULL,
+    code character varying(255),
+    created_by character varying(255),
+    created_time bigint,
+    is_nodal boolean,
+    last_modified_by character varying(255),
+    last_modified_time bigint,
+    name character varying(255),
+    parent character varying(255),
+    tenant_id character varying(255),
+    CONSTRAINT department_pkey PRIMARY KEY (id)
+);
+
+CREATE TABLE expenditure
+(
+    id character varying(255)  NOT NULL,
+    code character varying(255),
+    created_by character varying(255),
+    created_time bigint,
+    department_id character varying(255),
+    last_modified_by character varying(255),
+    last_modified_time bigint,
+    name character varying(255),
+    tenant_id character varying(255),
+    type character varying(255),
+    CONSTRAINT expenditure_pkey PRIMARY KEY (id)
+);
+
+CREATE TABLE project
+(
+    id character varying(255)  NOT NULL,
+    code character varying(255),
+    created_by character varying(255),
+    created_time bigint,
+    expenditure_id character varying(255),
+    last_modified_by character varying(255),
+    last_modified_time bigint,
+    name character varying(255),
+    tenant_id character varying(255),
+    CONSTRAINT project_pkey PRIMARY KEY (id)
+);
+
+
+
+CREATE TABLE project_department_entity_relationship
+(
+    project_id character varying(64) NOT NULL,
+    department_entity_id character varying(64),
```

Comment:
Do we need this extra table? 
Can it just be a single attribute in the project table? It will get overwritten in update api.

Reply:
Yes it's needed.